src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -189,14 +189,23 @@ private void RetrieveMatchingProcessesByProcessName()
                 if (!found &&
                     !WildcardPattern.ContainsWildcardCharacters(pattern))
                 {
+                    string errorText = ProcessResources.NoProcessFoundForGivenName;
+                    string errorName = "NoProcessFoundForGivenName";
+
+                    if(int.TryParse(pattern, out int x) && x > 0)
```

Comment:
one more tiny nit! A space between `if` and `(` since that's what we do everywhere else in this file.
```suggestion
                    if (int.TryParse(pattern, out int x) && x > 0)
```